testdata/dnn/onnx/generate_onnx_models.py

```diff
@@ -740,3 +741,30 @@ def forward(self, x):
 model = NormL2()
 x = Variable(torch.randn(1, 2, 3, 4))
 save_data_and_model("reduceL2_subgraph", x, model)
+
+from torchvision.ops.misc import *
+n = 3
+model = FrozenBatchNorm2d(n)
+model.eval()
+input = Variable(torch.rand( 1, 3, 2, 4 ))
+save_data_and_model("frozenBatchNorm2d", input, model)
+
+class UpsampleUnfusedTwoInput(nn.Module):
+
+    def __init__(self):
+        super(UpsampleUnfusedTwoInput, self).__init__()
+        self.conv1 = nn.Conv2d(3, 3, kernel_size=1, stride=1, padding=0)
+        self.conv2 = nn.Conv2d(3, 3, kernel_size=1, stride=1, padding=0)
+
+    def forward(self, x, y):
+        x = self.conv1(x)
+        x = x.shape[-2:]
+        y = self.conv2(y)
+        y = F.interpolate(y, size=x, mode="nearest")
```

Comment:
What if we will swap x and y so first input will be source but second used for shapes? Our PR works in that scenario?

Reply:
As there is no longer subgraph fusion for `UpsampleUnfusedTwoInputs` subgraph. Each node for that subgraph is enabled. so it doesn't matter swapping x and y.